stream-transformation-test/stream-transformation-performance-test/src/test/java/uk/gov/justice/framework/tools/transformation/StreamTransformationPerformanceIT.java

```diff
@@ -36,14 +39,16 @@ public void setupDatabase() throws Exception {
 
     }
 
-    @Ignore("Disabled by default, remove Ignore to run performance test")
+    //@Ignore("Disabled by default, remove Ignore to run performance test")
```

Comment:
do we want this ignored or commented out?

Reply:
ignored for IT test  removed the //